docs/library/machine.RTC.rst

```diff
@@ -75,6 +75,15 @@ Methods
       - ``wake`` specifies the sleep mode from where this interrupt can wake
         up the system.
 
+.. method:: RTC.memory([bytes])
+
+   Read or write the data of RTC memory.
```

Comment:
This is really useful, though I think it would be more helpful to micropython users to add some explanation of what RTC memory is and why it is useful, eg. something like:

```
Data written to RTC user memory is persistent across restarts, including `machine.soft_reset()` 
and `machine.deepsleep()`. ``RTC.memory(data)`` will write ``data`` to the RTC memory, where 
``data`` is any object which supports the buffer protocol (including `bytes`, `bytearray` and 
`array.array`). `RTC.memory()` reads RTC memory and returns a `bytes` object. 
The maximum length of RTC user memory is 2048 bytes by default.
```

Reply:
Thank you. done.